pages/service.js

```diff
@@ -93,6 +97,20 @@ const Service = ({Data, fontsLoaded}) => (
 					))}
 				</WorkOverview>
 
+				<UpdateLinks>
+					{ updates.map((update, index) => {
+						if (index < 4) {
+							return (
+								<UpdateLink
+									key={index}
+									title={update.title}
+									author={update.author.name}
+									date={getDateFormat(update.date)} />
+							);
+						}
```

Comment:
maybe we should think of some fallback if there are more items than for or do you think that is not important right now?